backend/app/app.server.js

```diff
@@ -2,66 +2,61 @@ import express from "express";
 import cookieSession from "cookie-session";
 import bodyParser from "body-parser";
 import path from "path";
-import GoogleCredentialController from "./controllers/google.credentials.controller";
+import morgan from "morgan"
+
 import fetchRooms from "./controllers/rooms.controller";
 import assets from "./controllers/assets.controller";
 import routes from "./app.routes";
-import auth from "./app.auth";
+
+import { setupAppAuth } from "./services/auth";
+
 import {
-  ROOMS_SOURCE,
-  ENVIRONMENT,
-  GOOGLE_CREDENTIAL,
-  ENFORCE_SSL,
-  CUSTOM_STYLE,
-  COOKIE_SESSION_SECRET,
-  COOKIE_SESSION_MAX_AGE,
+  getRoomsSource,
+  getCustomStyle,
+  getSessionConfig,
+  getEnvironment,
+  shouldEnforceSSL
 } from "./app.config";
 
 const app = express();
 
-app.locals.CUSTOM_STYLE = CUSTOM_STYLE;
+app.locals.CUSTOM_STYLE = getCustomStyle();
```

Comment:
I think the variable CUSTOM_STYLE is no longer necessary.
https://github.com/ResultadosDigitais/matrix/pull/167/files
Or maybe is missing feature in Morpheus layout

Reply:
🔪

Reply:
Yes... we can remove this variable :) my fault :disappointed: